lib/iris/tests/integration/plot/test_plot_2d_coords.py

```diff
@@ -63,5 +69,29 @@ def test_2d_coord_bounds_northpolarstereo(self):
         self.check_graphic()
 
 
+@tests.skip_plot
+@tests.skip_data
```

Comment:
@pp-mo you shouldn't need this decorator as you're not making use of external test data.

Reply:
:+1: cut+paste error, I think.
Good spot !!

Reply:
Ah yes, that old chestnut 🌰